ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-access-lists.j2

```diff
@@ -0,0 +1,19 @@
+{% if ip_access_lists is arista.avd.defined %}
+
+## IP Access-lists
+
+### IP Access-lists Summary
+
```

Comment:
Please move these blank line below if and else condition. So if anyone adds their snippet to this template in future, the consistency will be maintained.